src/HomePage.js

```diff
@@ -1,20 +1,111 @@
 import React from 'react';
 
 import 'bootstrap/dist/css/bootstrap.min.css';
+import { Row, Col, Button, Card, Container, Accordion, Toast } from 'react-bootstrap';
+import NavBar from './Components/NavBar';
+import './Styles/HomePage.css'
 
 class HomePage extends React.Component {
-  // Un-comment this for week 4 content 
-  // constructor(props){
-  //   super(props);
+  // Un-comment this for week 4 content
+
+  constructor(props){
+    super(props);
+
+    this.state = {
+      showToast : false
+    };
+
+    this.toggleShowToast = this.toggleShowToast.bind(this);
+
+  }
+
+  toggleShowToast(){
+    if(this.state.showToast == true){
```

Comment:
As shown in the security check, you should use === instead of == here for syntax. Or, `if(this.state.showToast)` will actually be a better way of writing this.